saleor/graphql/account/mutations/account/account_address_create.py

```diff
@@ -75,6 +80,29 @@ def perform_mutation(  # type: ignore[override]
                 utils.change_user_default_address(user, address, address_type, manager)
         return AccountAddressCreate(user=user, address=address)
 
+    @classmethod
+    def get_invalid_fields(cls, cleaned_input):
+        invalid_fields = []
+        for field, value in cleaned_input.items():
+            if not value and field not in (
```

Comment:
I'm not sure if this is really correct way to handle the issue. I would rather expect that there should be problem within `clean` method where validation of the mentioned field in issue is happening.
Didn't we also allow providing e.g empty `firstName` before, is this expected to remove such possibility? @maarcingebala

Reply:
I'm not sure if I follow. The bug is happening in core code, as pointed in raised issue, what does frontend have to do with this? And what package exactly you want to introduce?

My concern is, that this code here feels hacky (e.g. you change behaviour of `firstName` field), and I'm not sure if this is right place for fix. Did you check what is happening within the mixture for address validation?

Reply:
@kadewu, I apologise if my previous response was unclear. I wanted to better understand the issue by exploring the frontend validation and data formatting aspects.

I acknowledge that the bug is occurring in the core code, specifically within the clean method responsible for address validation. I understand your concern that modifying the behaviour of the "firstName" field may seem like a quick fix rather than addressing the root cause of the issue.

I appreciate your suggestion to investigate the clean method for a more appropriate fix. I will focus on analysing that part of the code to identify any problems or potential improvements. 

Thank you for your patience and valuable input regarding this issue. 

Please feel free to share any additional questions or suggestions you may have regarding this matter.

Reply:
No need to apologize, cause nothing happened. ;)

Just side note, it may happen that fixing it inside mentioned method may not be possible, please also remember that the `I18nMixin` is used by other mutations.